google/resource_compute_subnetwork.go

```diff
@@ -142,18 +148,32 @@ func resourceComputeSubnetworkCreate(d *schema.ResourceData, meta interface{}) e
 	}
 
 	// Build the subnetwork parameters
-	subnetwork := &compute.Subnetwork{
+	subnetwork := &computeBeta.Subnetwork{
 		Name:                  d.Get("name").(string),
 		Description:           d.Get("description").(string),
 		IpCidrRange:           d.Get("ip_cidr_range").(string),
 		PrivateIpGoogleAccess: d.Get("private_ip_google_access").(bool),
-		SecondaryIpRanges:     expandSecondaryRanges(d.Get("secondary_ip_range").([]interface{})),
+		SecondaryIpRanges:     expandSecondaryRangesV0Beta(d.Get("secondary_ip_range").([]interface{})),
 		Network:               network.RelativeLink(),
+		EnableFlowLogs:        d.Get("enable_flow_logs").(bool),
 	}
 
 	log.Printf("[DEBUG] Subnetwork insert request: %#v", subnetwork)
 
-	op, err := config.clientCompute.Subnetworks.Insert(project, region, subnetwork).Do()
+	subnetworkV1 := &compute.Subnetwork{}
```

Comment:
What're your thoughts on moving this struct creation into [here](https://github.com/terraform-providers/terraform-provider-google/pull/1385/files#diff-c00013899b923b404c727949952f35ffR163) as well as everything specific to subnetworkV1?

Reply:
Was that supposed to link to https://github.com/terraform-providers/terraform-provider-google/pull/1385/files#diff-c00013899b923b404c727949952f35ffR172 ?

The reason I didn't do that is because I need the struct accessible so I can generate the ID using it.